### PR TITLE
[TASK] Align with new TYPO3 documentation standards

### PR DESCRIPTION
**Pull request by alexander-nitsche**
Relates: https://github.com/TYPO3-Documentation/T3DocTeam/issues/182

### CODE REVIEW

**Comment by alexander-nitsche** [comment]
What is missing: 
(1) Registering this documentation for rendering and publishing at docs.typo3.org with the [webhook](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingDocForExtension/Webhook.html#github).
(2) Adding the new manual URL `https://docs.typo3.org/p/friendsoftypo3/compatibility7/main/en-us/` to the TER extension page configuration.

**Comment by alexander-nitsche** [comment]
This is how it looks like:
Screen Shot 2022-04-01 at 19 00 00

**Comment by alexander-nitsche** [comment]
Added two new commits regarding badges in README and additional sources (TER, repository and docs.typo3.org) in composer.json.

In addition, the TER homepage is missing the "Code Insights" and "Found an issue?" buttons, which seem to be caused by the empty "Link to repository" and "Link to issue tracker" fields in the TER extension edit form.

Relates: TYPO3-Documentation/T3DocTeam#185

File: Documentation/Index.rst [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+The extension is published under GPL v2+, all included third-party libraries are
+published under their respective licenses.
+
+Authors
+-------
+
+A lot of contributors have been working on this area while this functionality was part of
+the TYPO3 Core. This package is now maintained by a loose group of TYPO3 enthusiasts inside
+the TYPO3 Community. Feel free to contact `Benni Mack`_ for any questions
```
Comment (ohader) [inline Documentation/Index.rst]: Please avoid pointing to Benni here, refer to the issue tracker instead...

**Review by ohader — CHANGES_REQUESTED** [review]
Avoid author pointing

File: Documentation/Index.rst (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/Index.rst]: Done.

**Review by ohader — APPROVED** [review]

**Comment by ohader** [comment]
Adjusted details in TER https://extensions.typo3.org/extension/compatibility7

**Pull request merged**